src/components/App.js

```diff
@@ -10,11 +10,13 @@ export default class App extends React.Component {
 
   render() {
     return (
-      <div className="app-panel">
-        <h1>Hello from App</h1>
-        <Display />
-        <ButtonPanel />
-      </div>
+      <>
+        <div className="app-panel">
```

Comment:
## Changes:

#### ADD Flex display mode for these components and other missing features for this milestone.

- [ ] Please add flex display mode for this component, the `Display` and `ButtonPanel` components ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/useFlex.png))

- [ ] The calculator width needs to strickly equal 700px as instructed in the requirements ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/700Wide.png)).
 ##

#### `Display` component

* [ ] As instructed in this milestone, this component needed to have a grey background. Let's implement this. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-bg.png)). 
* [ ] It needs also, to strickly have a height of 100px ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-height.png)).
* [ ] While at it, remember to also add a reasonable padding from the edges ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-padding.png)).
##
#### `ButtonPanel` component
* [ ] Ensure that each row's height is 100px ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/rowHeight.png)).
##

#### `Button` component
- For this component: 👇 
* [ ] Ensure that buttons take 25% of container width (except `0` button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/25Wide.png)).
* [ ] Ensure that buttons have borders ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/ButtonsBoarder.png)).
* [ ] Ensure that buttons text is centered and black ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/TextBlack.png)).
* [ ] Ensure that it accepts `color` and `wide` as an incoming props ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png)).
* [ ] Ensure that the default `color` for button is orange ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)).
* [ ] Ensure that `wide` is boolean, true means that button take 50% of space (used for `0` button) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png)).
##
### Please note:
**These changes are mandatory only if you are not using the `v1.1` of React, [Milestone description here](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106475924/).**
##

Reply:
Hello man, although I agree with most of the changes required, the point of this whole Milestone is just a reminder to not waste any time styling it so far. All the requirements you have provided me now are for the final version of the project. 
At the edited description for the pull request, you can read the description from the curriculum.

Reply:
Hello, @rsnazario Apologies for any confusion this review must have cased you. I understand you are using v1.1 and this milestone ought to be approved without the styling bit. 

I wish you all the very best in your next tasks. Cheers!

### Happy coding 💻 

#### Christian Otieno (TSE)

Reply:
No problem man, I understand that you were aiming for my best result and I appreciate it :) 
It also helps for the next milestones hahaha
I hope you have a great weekend